statefulsyncer/configuration.go

```diff
@@ -66,3 +68,9 @@ func WithSeenConcurrency(concurrency int64) Option {
 		s.seenSemaphoreSize = concurrency
 	}
 }
+
+func WithExtraSyncerOpts(os ...syncer.Option) Option {
+	return func(s *StatefulSyncer) {
+		s.extraSyncerOpts = os
+	}
+}
```

Comment:
A more flexible solution would be to pass a mapping function like in `WithCustomHelper` below. But that's probably unnecessarily complex.